<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin::time_zone::SettingsWidget</class>
 <widget class="QWidget" name="plugin::time_zone::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Zone</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time Zone Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="str_format_edit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Z (O)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">C (O)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">n</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">a</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="str_format_help_btn">
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>159</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
